### ISSUE TITLE
NTR: Elevated urinary cytidine

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4944]
**Preferred term label:**
Elevated urinary cytidine

**Synonyms**
Elevated cytidine in urine

**Definition (free text, please give PubMed ID)**
Increased levels of urinary cytidine, a pyrimidine nucleoside in which cytosine is attached to ribofuranose via a beta-N1-glycosidic bond.

Source: CHEBI:17562

**Parent term (use hpo.jax.org/app)**
HP_0031979 'Abnormal urine carbohydrate level'

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:618477 

See ticket: https://github.com/monarch-initiative/hpo-annotation-data/issues/387

**Comment by pnrobinson** [comment]
added as HP_0032573
